R/deployApp.R

```diff
@@ -85,13 +87,54 @@ deployApp <- function(appDir = getwd(),
                       upload = TRUE,
                       launch.browser = getOption("rsconnect.launch.browser",
                                                  interactive()),
-                      quiet = FALSE,
+                      logLevel = c("normal", "quiet", "verbose"),
                       lint = TRUE,
                       metadata = list()) {
 
   if (!isStringParam(appDir))
     stop(stringParamErrorMessage("appDir"))
 
+  # respect log level
+  logLevel <- match.arg(logLevel)
```

Comment:
I assume "normal" is the default logLevel; is that set explicitly somewhere, or is it de facto?